src/app/(marketing)/components/DocsSidebar.tsx

```diff
@@ -12,6 +12,8 @@ export default function DocsSidebar({ featureSets }: FeatureSetProps) {
   const pathname = usePathname();
   const [isOpen, setIsOpen] = useState(false);
 
+  const activeFeatureSets = featureSets.filter((featureSet) => featureSet.features && featureSet.features.length > 0);
+
```

Comment:
Not important (not necessary to change it) but could just be 
`featureSet?.features?.length` instead of `featureSet.features && featureSet.features.length > 0` (length 0 is falsy)